.gitmodules

```diff
@@ -10,3 +10,6 @@
 	path = clang
 	url = https://github.com/RadeonOpenCompute/hcc-clang-upgrade.git
 	branch = clang_tot_upgrade
+[submodule "compiler-rt"]
```

Comment:
@kknox from checkout speed perspective, do you think moving to repo bring any benefit?